### DISCUSSION TITLE
Apache Parquet import

### DISCUSSION THREAD

**Discussion by vbmithr** [GreptimeTeam/greptimedb#883] [Ideas]
Must have :) DuckDB and Clickhouse have it (though clickhouse currently is stuck with fixed_length_byte_array column types…)

**Comment by killme2008** [comment]
@vbmithr  Agree, we should add this feature. Maybe it's a part of greptimedb command-line utilities, we already created an issue to track it #454 .

But I think we can create an independent issue for bulk loading data. What do you think? @fengjiachun @evenyag @waynexia

**Reply by waynexia** [reply]
There is also an issue for bulk load https://github.com/GreptimeTeam/greptimedb/issues/405

We can support parquet first. It's our storage format, and looks like there are not very complex things need to do to support it. Verifying schema, adding our metadata, and registering the new table, etc should be enough.
